todo_app/templates/index.html

```diff
@@ -11,7 +11,7 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        {% for item in to_do_items %}
+        {% for item in view_model.items %}
```

Comment:
Worth noting that the exercise (not very clearly admittedly) assumes we'll pivot the front end to actually use the `to_do_items` and `done_items` properties - e.g. by turning this into two loops, one over each category - rather than just swapping this out for the `items` property. I'm not very fussed either way, but as long as we're conscious that this approach means the unit tests are very much for show since they're testing something we never actually use!